scripts/create_policy

```diff
@@ -285,6 +285,15 @@ def _get(url):
         return None
 
 
+def _get_filelists_ext_xml(repo, repomd_xml):
+    root = ET.parse(repomd_xml).getroot()
+    location = root.find(
+        "./{http://linux.duke.edu/metadata/repo}data[@type='filelists-ext']/{http://linux.duke.edu/metadata/repo}location"
+    )
+    if location is not None:
+        return urllib.parse.urljoin(repo, location.attrib["href"])
+
```

Comment:
Do we want to explicitly return None here?

Reply:
Right, the return None is implicit, I will make it explicit

Reply:
Done